recipes/taglib/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class TaglibConan(ConanFile):
+    name = "taglib"
+    description = "TagLib is a library for reading and editing the metadata of several popular audio formats."
+    license = ("LGPL-2.1-or-later", "MPL-1.1")
+    topics = ("conan", "taglib", "audio", "metadata")
+    homepage = "https://taglib.org"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "bindings": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "bindings": True,
+    }
+
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake", "cmake_find_package"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("zlib/1.2.11")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["ENABLE_CCACHE"] = False
+        self._cmake.definitions["VISIBILITY_HIDDEN"] = True
+        self._cmake.definitions["BUILD_TESTS"] = False
+        self._cmake.definitions["BUILD_EXAMPLES"] = False
+        self._cmake.definitions["BUILD_BINDINGS"] = self.options.bindings
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("COPYING.*", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), "taglib-config")
```

Comment:
Do we need to drop empty leftover directories (`bin`)?

Reply:
Not for windows & shared

Reply:
That's ok. Empty dirs are not tracked by the Conan manifest and therefore not created while installing from remote.